### DISCUSSION TITLE
making installation public steps still functional?

### DISCUSSION THREAD

**Discussion by GoodyTheGoodman** [mcguirepr89/BirdNET-Pi#867] [Q&A] [answered]
After a long cold winter with very little bird activity, I have booted the pi back ip with BirdNET-Pi.  I can't wait to get things running!  I am currently having issues making the installation public.  I own a domain that I would like to use but cannot get it configured for some reason. 

Like last time, I created a DHCP reservation, configured port forwarding, registered 1 Hostname with NoIP.com, registered 1 DNS CNAME Record, changed my BirdNET-Pi Advanced Settings, and installed the NoIP DUC. 

 Has anyone else been having issues? If so do you have any advice for getting my website running publicly on my domain name? I have referred to similar discussions and nothing seemed to help. 
 
If there is a less secure way of making it remotely accessible that would be fine.  The browser will be fullscreen with no peripherals attached.  It will act only as a large screen dashboard for people to see current activity in my home.  The reason I need it accessible publicly is because I run my Pi on a separate network from my home network, given that it's placed remotely on our property.

**Comment by ehpersonal38** [comment, marked as answer]
Hi @johnocampbell99 

I'm not too familiar with the particulars of the Wiki's steps for making your installation public as I've never attempted it before, though I can definitely help you out with the "less secure way of making it public" that you were open to. Here is a basic outline of what you need to do (I may have ncluded steps you've already done, just ignore those):

1. Port forward port 80 of your Pi's IP address using your router's web interface.
2. On your BirdNET-Pi web interface, go to Tools -> Settings -> Advanced Settings. Find the "Custom URL" settings option, and change the value to the domain of the website you're intending to use. Be sure there is NOT a trailing slash on the URL! 
Example:
![chrome_0isyfqJoGn](https://user-images.githubusercontent.com/103586016/233807228-abd4d3eb-260a-4d51-8cc9-dcfcc2d8c2c8.png)
3. Click "Update Settings" at the bottom of the page.
4. Now you will need to add an "A" record to your domain that points at your public IP address (search Google for "my ip" to get this value).
Example:
![chrome_0J2fToPLt1](https://user-images.githubusercontent.com/103586016/233807392-f2cc7c07-1db7-414c-a1b4-4f913e1d614d.png)
5. Save your domain settings. This record may take a while to propogate around the world, you can search online for a "DNS propogation checker" tool to verify when the record propogates successfully.

If all has been done well, you'll be able to access your BirdNET-Pi publically at the Custom URL you set up in step 2. For me this means I can now access my BirdNET-Pi at
`http://birds.mydomain.com`

Hope this helped!

**Reply by GoodyTheGoodman** [reply]
Thank you!!! This has been a huge help.

**Reply by NilsEinsfelder** [reply]
I had to additionally forward port 443 for this solution to work. 

Thanks for the Explanation, would be great to add this or a link to this to the [Sharing Guide](https://github.com/mcguirepr89/BirdNET-Pi/wiki/Sharing-Your-BirdNET-Pi).
